internal/quickfort/zone.lua

```diff
@@ -105,9 +105,9 @@ local zone_db_raw = {
     b={label='Bedroom', default_data={type=df.civzone_type.Bedroom}},
     h={label='Dining Hall', default_data={type=df.civzone_type.DiningHall}},
     n={label='Pen/Pasture', default_data={type=df.civzone_type.Pen,
-       assign={zone_settings={pen={unk=1}}}}},
+       assign={zone_settings={pen={check_occupants=true}}}}},
     p={label='Pit/Pond', props_fn=parse_pit_pond_props, default_data={type=df.civzone_type.Pond,
-       assign={zone_settings={pit_pond=df.building_civzonest.T_zone_settings.T_pit_pond.top_of_pit}}}},
+       assign={zone_settings={pond={flag={keep_filled=true}}}}}},
```

Comment:
Strictly speaking, the old `top_of_pit` value should be equivalent to `check_occupants=True` and `keep_filled=False`.